src/components/include/policy/policy_regular/policy/policy_manager.h

```diff
@@ -785,6 +792,18 @@ class PolicyManager : public usage_statistics::StatisticsManager,
   virtual AppIdURL RetrySequenceUrl(const struct RetrySequenceURL& rs,
                                     const EndpointUrls& urls) const = 0;
 
+  /**
+   * @brief OnLocalAppAdded triggers PTU
+   */
+  virtual void OnLocalAppAdded() = 0;
```

Comment:
@kshala-ford All web engine app are considered to be local. 
This terminology is used in the original proposal.
Please see: 
1) https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0240-sdl-js-pwa.md#22-activating-a-web-app
`Activating a local web app...`
2) https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0240-sdl-js-pwa.md#24-hmi-api-using-app-properties-rpcs
`For local apps ...`
3) https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0240-sdl-js-pwa.md#impact-on-existing-code
`App registration on SDL Developer Portal (smartdevicelink.com) needs a new field for local apps.`
and in other places as well. 
We have 2 reasons to use this terminology:
- It is used in the related proposal
- As `WebApp` was used in Cloud Transport Adapter proposal - we would like to avoid confusion.

Reply:
Thanks @AByzhynar that is helpful :)

Reply:
👍